context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -78,7 +79,11 @@ def get_conf_cells(self):
                     )
                 )
             dataset = dataset.add_object(MultiImageWrapper(image_wrappers))
-            vc = self._setup_view_config_raster(vc, dataset)
+            vc = self._setup_view_config_raster(
+                vc,
+                dataset,
+                disable3d=[self._get_hybcycle(img_path) for img_path in sorted_images]
```

Comment:
```suggestion
                disable_3d=[self._get_hybcycle(img_path) for img_path in sorted_images]
```
but beyond that, it's really not clear to me what this parameter does... Thought it was a boolean at first. I'd like to use nouns for things / verbs for functions.

Reply:
It's just a pass-through prop to the vitessce layer.  The camel-case thing can be fixed though in the Python.

Reply:
It allows us to "turn off" the option for 3D for seqfish and CODEX because we do not want to allow users to enter 3D mode for those assays.